src/main/java/cs2103/v15_1j/jimjim/command/SaveLocationCommand.java

```diff
@@ -21,7 +23,28 @@ public String getSavePath() {
     @Override
     public UIFeedback execute(ControllerStates conStates) {
     	String backup = conStates.config.savePath;
+    	File file = null;
+    	
+    	if (savePath.contains(" ")) {
```

Comment:
then "abc\ def" should be allowed right but you are not allowing it if you have this condition